src/lib/webtab/webtab.cpp

```diff
@@ -292,6 +295,24 @@ void WebTab::setPinned(bool state)
     m_isPinned = state;
 }
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,7,0)
+bool WebTab::isMuted() const
+{
+    return m_webView->page()->isAudioMuted();
+}
+
+void WebTab::setMuted(bool muted)
+{
+    m_webView->page()->setAudioMuted(muted);
```

Comment:
And this. So just #ifdef the code that cannot be built with Qt 5.7 (= just call to new QWebEnginePage functions/signals).